eslint.config.js

```diff
@@ -3,21 +3,17 @@ import {fileURLToPath} from "node:url"
 
 import {includeIgnoreFile} from "@eslint/compat"
 import js from "@eslint/js"
-import prettier from "eslint-config-prettier"
 import jsxA11y from "eslint-plugin-jsx-a11y"
 import react from "eslint-plugin-react"
 import reactHooks from "eslint-plugin-react-hooks"
 import simpleImportSort from "eslint-plugin-simple-import-sort"
-import globals from "globals"
```

Comment:
To my knowledge, all the core ESLint rules that use `globals` are superseded by [typescript-eslint extension rules](https://typescript-eslint.io/rules/#extension-rules). I don't know of any plugin rules that use `globals` themselves.

Reply:
I was using `globals` to be explicit when I defined the supported environments in `languageOptions`.

```
globals: {
    ...globals.browser,
    ...globals.node,
    ...globals.es2025,
    ...globals.jest,
},
```

Reply:
Huh, just double checked this and you're right, `globals` has no effect. Weird...